inscriptions/ISic030010.xml

```diff
@@ -109,7 +115,7 @@
                                 <placeName type="modern">Selinunte</placeName>
```

Comment:
`<placeName type="ancient" ref="http://pleiades.stoa.org/places/462489">Selinus</placeName>		
                                <placeName type="modern" ref="http://sws.geonames.org/2523164">Archaeological site of Selinunte</placeName>
                                <geo>37.58337, 12.82522</geo>`